pkg/tc/classes.go

```diff
@@ -13,15 +13,15 @@ import (
 	"github.com/florianl/go-tc"
 )
 
-const (
-	QFQHelp = `Usage: ... qfq weight NUMBER maxpkt BYTES`
-
-	maxUint32 = 0xFFFF_FFFF
-)
+const maxUint32 = 0xFFFF_FFFF
```

Comment:
there is a constant math.MaxUint32, but maybe it smart move to reduce binary size?